client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -101,13 +101,33 @@ class DomainManagementNavigationEnhanced extends React.Component {
 			return null;
 		}
 
-		// NOTE: remember to add translate to the description string once you start working on it
+		const { wpcom_url: wpcomUrl } = selectedSite;
```

Comment:
I'm not sure `wpcom_url` is always set. In my case this is leading to weird results. Check the main comment

Reply:
Good catch - my test Atomic site was not using custom domain and I totally missed that one